File: VB/SetSelection/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace SetSelection
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnDiagram_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnDiagram.Click
			If Not Me.btnDiagram.IsHandleCreated Then Return

			' Select a diagram.
			chartControl1.SetObjectSelection(chartControl1.Diagram)
		End Sub

		Private Sub btnAxis_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnAxis.Click
			If Not Me.btnAxis.IsHandleCreated Then Return

			' Select an axis.
			chartControl1.SetObjectSelection(CType(chartControl1.Diagram, XYDiagram).AxisX)
		End Sub

		Private Sub btnSeriesPoint_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnSeriesPoint.Click
			If Not Me.btnSeriesPoint.IsHandleCreated Then Return

			' Select a series point.
			Dim pointToSelect As SeriesPoint = TryCast(Me.chartControl1.Series(0).Points(1), SeriesPoint)
			If pointToSelect IsNot Nothing Then
				Me.chartControl1.SetObjectSelection(pointToSelect)
			End If
		End Sub
	End Class
End Namespace

File: CS/SetSelection/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace SetSelection {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void btnDiagram_Click(object sender, EventArgs e) {
            // Select a diagram.
            chartControl1.SetObjectSelection(chartControl1.Diagram);
            chartControl1.Refresh();
        }

        private void btnAxis_Click(object sender, EventArgs e) {
            // Select an axis.
            chartControl1.SetObjectSelection(((XYDiagram)chartControl1.Diagram).AxisX);
            chartControl1.Refresh();
        }

        private void btnSeriesPoint_Click(object sender, EventArgs e) {
            // Select a series point.
            SeriesPoint pointToSelect = this.chartControl1.Series[0].Points[1] as SeriesPoint;
            if (pointToSelect != null) {
                this.chartControl1.SetObjectSelection(pointToSelect);
            }
            chartControl1.Refresh();
        }
    }
}
